.travis.yml

```diff
@@ -1,13 +1,41 @@
 sudo: false
+
 language: python
+
 cache: pip 
+
 python:
   - 3.5
+
+env:
+  - PYDATA=true
+  - PYDATA=false CHECK_MIGRATION=true
+
+before_install:
+  - git config --global user.email "you@example.com"
+  - git config --global user.name "Your Name"
+  - if [[ $TRAVIS_PULL_REQUEST != false ]]; then
+      git pull -q --no-edit origin develop;
+    fi;
+  - if [[ $PYDATA == true ]]; then
+      git pull -q --no-edit origin deploy-pydata;
```

Comment:
>Why not conditionally include it in the config? It could be controlled via envvar.

Nice suggestion. I am starting to reconsider #938. By overriding the envvar `DJANGO_SETTINGS_MODULE`, we can switch between a normal deployment and a PyData deployment.

>So… is this what @chrismedrela intended in #1066?

Please refer to the discussion at #1066. As you pointed out, Travis does merge the develop branch before testing.